### DISCUSSION TITLE
Duplicate Presets

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18914] [Feature Requests]
### Summary

For instance, when multiple presets on a same collection differ only by the filter, it would be nice just to Copy / Duplicate the preset and just modify the filter avoiding to have to reformat the layout.

### Basic Example

Have a preset and copy it.

### Motivation

avoiding to have to reformat the same features: layout, etc.

### Detailed Design

Add a Copy button on top of the preset
and ask for a new name

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
